src/components/clusters/ClusterPage.tsx

```diff
@@ -108,7 +108,7 @@ const ClusterPage: React.FC<RouteComponentProps<MatchParams>> = ({ match }) => {
 
   if (uiState === ResourceUIState.ERROR) {
     if (errorDetail?.code === '404') {
-      return <Redirect to="/clusters" />;
+      return <Redirect to={`${routeBasePath}/clusters`} />;
```

Comment:
This fix raises question if we want to redirect to the assisted-installer list and so skipping the "standard" 404 page of the OCM. Wdyt @jtomasek ?

Reply:
I think this is a good fix for now and we can revisit when the cluster lists get merged